tests/import_test.go

```diff
@@ -135,6 +135,29 @@ var _ = Describe("[rfe_id:1115][crit:high][vendor:cnv-qe@redhat.com][level:compo
 			Expect(f.GetDiskGroup(f.Namespace, pvc, false)).To(Equal("107"))
 		}
 	})
+
+	It("Import registry disk into PVC barely big enough", func() {
+		imageURL := fmt.Sprintf(utils.LargePhysicalDiskQcow, f.CdiInstallNs)
```

Comment:
What is the size of the Image? What does it mean "barely big enough"?

Reply:
The image is found in tests/images/cirros-large-physical-size.qcow2, and its virtual size is 2Gi. What barely big enough means, I am not sure either. The requested PVC has 12% overhead so should be fine.

Reply:
The truth is, I don't remember the intention behind the test. Maybe we should reconsider?